sim/xcelium/__init__.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python3
+
+# HAMMER-VLSI PLUGIN, XCELIUM 
+# Notes: This plugin sets up xrun to execute in a three-step xrun invocation.
+#        This bridges multi-tool direct-invocation and the xrun single-invocation.
+
+
+import os
+import re
+import shutil
+import json
+import datetime
+import io
+import sys
+from typing import Dict, List, Optional, Callable, Tuple
+from multiprocessing import Process
+
+import hammer_utils
+import hammer_tech
+from hammer_tech import HammerTechnologyUtils
+from hammer_vlsi import FlowLevel, TimeValue
+from hammer_vlsi import HammerSimTool, HammerToolStep, HammerLSFSubmitCommand, HammerLSFSettings
```

Comment:
You haven't hooked into LSF settings here, but this brings up the question - is there a method to control # of cores used for simulation (hopefully better than the buggy VCS implementation)?

Similarly, are you able to run parallel simulations based on the `sim.inputs.benchmarks` and `sim.inputs.parallel_runs` settings?

Reply:
I've hooked in LSF, and multicore support. Will need to revisit to provide more explicit hammer warnings for unsupported tcl commands, options, etc. but will hold off until license is obtained since my testing goes only up to elaboration.

I think I should be able to get parallel sims, but not super familiar with how to test that. Could potentially PR that in later? Maybe like a proper multicore + parallel support PR sometime early jan.